modules/optimization/doc/content/modules/optimization/examples/shapeOpt_Annulus.md

```diff
@@ -0,0 +1,122 @@
+# Constrained Optimization Example: Shape Optimization
+
+## Background
```

Comment:
I would suggest having a problem description section before jumping into the example. Explain the geometry, the physics, and what is trying to be optimized.